storage/src/main/java/org/apache/uniffle/storage/util/ShuffleStorageUtils.java

```diff
@@ -199,7 +200,7 @@ public static int[] getPartitionRange(int partitionId, int partitionNumPerRange,
 
   public static int getStorageIndex(int max, String appId, int shuffleId, int startPartition) {
     String hash = appId + "_" + shuffleId + "_" + startPartition;
-    int index = MurmurHash.getInstance().hash(hash.getBytes()) % max;
+    int index = MurmurHash.getInstance().hash(hash.getBytes(StandardCharsets.UTF_8)) % max;
```

Comment:
Why do we need change this? This string only contain ASCII code.

Reply:
The method `getBytes()` encodes a String into a byte array using the platform's default charset if no argument is passed. This will cause the application behavior to vary between platforms.

The default charset is determined during virtual-machine startup and typically depends upon the locale and charset of the underlying operating system.

If you think `UTF-8` is a overkill here, we can change it to `US_ASCII`.

Reply:
`US_ASCII` will be better. It's strange to use `UTF-8` here.

Reply:
Encoding a string with only `ASCII` text in `UTF-8` is same as in `ASCII`.
So it's not necessary to change it.

```java
  @Test
  public void testEncoding() {
    assertArrayEquals("123_456_789".getBytes(StandardCharsets.UTF_8),
        "123_456_789".getBytes(StandardCharsets.US_ASCII));
  }
```

Reply:
OK